static/js/components/WidgetWrappers.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react'
+import Octicon from 'react-component-octicons'
+
+
+
+class mitodlWidgetListWrapper extends Component {
+    render() {
+      return (
+        <div>
+          {this.props.renderList(this.props.widgetInstances)}
+        </div>
+      )
+    }
+}
+
+class mitodlWidgetWrapper extends Component {
```

Comment:
are the components written in this file used anywhere?

Reply:
Those are stubs but haven't been fully implemented yet. They also will probably be nicer if / when we implement redux in the package and then we won't need to pass through so many widget props.